lib/ocpp/common/websocket/websocket_tls_tpm.cpp

```diff
@@ -151,6 +151,93 @@ struct WebsocketMessage {
     std::atomic_bool message_sent;
```

Comment:
we propably want to rename these files `websocket_tls_tpm.cpp` and `websocket_tls_tpm.hpp` to `websocket_libwebsockets` or sth. similar because it supports TLS and non-TLS

Reply:
Makes sense, makes sense.